library/network/src/modules/SuSEFirewallServices.rb

```diff
@@ -337,12 +338,13 @@ def GetMetadataAgent(filefullpath)
 
     # Returns service definition.
     # See @services for the format.
-    # If `silent` is not defined or set to `true`, function throws an exception
-    # SuSEFirewalServiceNotFound if service is not found on disk.
+    # If *silent* is `false` (the default), the method throws an exception
```

Comment:
`true` -> `false` is a factual correction. The method is only used in this file so it was easy to check.